core/statistics.c

```diff
@@ -207,6 +226,14 @@ void calculate_stats_summary(struct stats_summary *out, bool selected_only)
 		prev_month = current_month;
 		prev_year = current_year;
 	}
+
+	/* add labels for depth ranges up to maximum depth seen */
+	for (dr=0; dr * 10000 <= out->stats_by_depth[0].max_depth.mm; ++dr)
+	{
+		asprintf(&label, "%dm - %dm", dr * 10, (dr+1) * 10);
```

Comment:
... and didn't we agree not to use that but to use our version?

Reply:
@dirkhh I think this was an outdate version of statistics.c. The next commit moved the label creation out of statistics.c and into qt-models/yearlystatisticsmodel.cpp where I could do away with the asprintf altogether and use the get_depth_string and get_temperature_string functions defined in qthelper.cpp to create QString objects.